functions/go/enable-gcp-services/metadata.yaml

```diff
@@ -0,0 +1,12 @@
+image: gcr.io/kpt-fn/enable-gcp-services
+description: Generate and manage project service enablements for a GCP project.
+tags:
+  - generator
+  - GCP
+  - mutator
+sourceURL: https://github.com/GoogleContainerTools/kpt-functions-catalog/tree/master/functions/go/enable-gcp-services
+examplePackageURLs:
```

Comment:
What do you think about moving the pruning e2e test to examples? IMO it'd be useful showing the users how pruning work in an example.

Reply:
moved to example